package.json

```diff
@@ -39,7 +39,6 @@
   },
   "devDependencies": {
     "chai": "4.1.0",
-    "dotenv": "4.0.0",
```

Comment:
@michaelmurray6298 Ah sorry dude, I should have been  more clear, this needs to be placed back into the `dependencies` object.